automation/run-tests.sh

```diff
@@ -283,8 +215,52 @@ function upload_coverage {
     fi
 }
 
+function check_iface_exist {
+    exec_cmd "ip a s | grep -q $1"
```

Comment:
`ip address` is not a good command for checking existence of NIC. Please use `ip link` instead.

Reply:
Done, thanks!